lib/blueprinter/helpers/type_helpers.rb

```diff
@@ -3,13 +3,19 @@
 module Blueprinter
   module TypeHelpers
     private
-    def active_record_relation?(object)
-      !!(defined?(ActiveRecord::Relation) &&
-        object.is_a?(ActiveRecord::Relation))
+
+    def array_like_classes
+      [
+        Array,
+        defined?(ActiveRecord::Relation) && ActiveRecord::Relation,
```

Comment:
if it's not defined this becomes `nil` and the `.compact` at the end gets rid of it.